manticore/utils/mappings.py

```diff
@@ -43,5 +43,38 @@ def _mmap(fd, offset, size):
     mmap = _mmap
     munmap = _munmap
 
+elif osname.startswith("netbsd"):
+
+    libc = ctypes.cdll.LoadLibrary("libc.so.12")
```

Comment:
why not `libc.so`?

Reply:
On some OSes (GNU/Linux?) we require `libc.so.$VERSION`, on NetBSD we prefer `libc.so`